deerlab/lsqcomponents.py

```diff
@@ -1,11 +1,14 @@
 import numpy as np
 
-def lsqcomponents(V,K,L,alpha,weights, regtype = 'tikh',huberparam = 1.35):
+def lsqcomponents(V,K,L=None,alpha=0,weights=1, regtype = 'tikh',huberparam = 1.35):
 # ==============================================================================================
 
     # Prepare
     nr = np.shape(K)[1]
 
+    if L is None:
+        L = np.eye(nr)

```

Comment:
Maybe set L to zero?

Reply:
If the `L=None` then the `lsqcomponents` function now returns directly the `AtA` and `AtV` components without the regularization terms.